app/views/project_instances/_form.html.erb

```diff
@@ -0,0 +1,77 @@
+<%= form_for(@project_instance, :html => {class: "form-horizontal"}) do |f| %>
+  <% if @project_instance.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@project_instance.errors.count, "error") %> prohibited this instance from being saved:</h2>
+
+      <ul>
+      <% @project_instance.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="form-group">
+    <%= f.label :project_id, class: "col-sm-2 control-label" %>
+    <div class="col-sm-10">
+    <%= f.collection_select :project_id, Project.order(:title),:id,:title, 
+        {prompt: "Select the project this Instance refers to"}, class: "form-control" %>
+      </div>
+  </div>
+
+  <div class="form-group">
+    <%= f.label :name, class: "col-sm-2 control-label" %>
+    <div class="col-sm-10">
+      <%= f.text_field :name, class: "form-control" %>
+    </div>
+  </div>
+
+  <div class="form-group">
+    <%= f.label :url, class: "col-sm-2 control-label" %>
+    <div class="col-sm-10">
+      <%= f.text_field :url, class: "form-control" %>
+    </div>
+  </div>
+
+  <div class="form-group">
+    <%= f.label :backup_information, class: "col-sm-2 control-label" %>
+    <div class="col-sm-10">
+      <%= f.text_field :backup_information, class: "form-control" %>
+    </div>
+  </div>
+
+  <div class="form-group">
+    <%= f.label :stage, class: "col-sm-2 control-label" %>
+    <div class="col-sm-10">
+      <%= f.select :stage, [["Staging", "Staging"],["Production", "Production"]], {}, class: "form-control" %>
+    </div>
+  </div>
+
+  <div class="form-group">
+    <%= f.label :branch, class: "col-sm-2 control-label" %>
+    <div class="col-sm-10">
+      <%= f.text_field :branch, class: "form-control" %>
+    </div>
+  </div>
+
+  <div class="form-group">
+    <%= f.label :description, class: "col-sm-2 control-label" %>
+    <div class="col-sm-10">
+      <%= f.text_area :description, class: "form-control" %>
+    </div>
+  </div>
+
+  <div class="form-group">
+    <%= f.label :installation_ids, class: "col-sm-2 control-label" %>
+    <div class="col-sm-10">
+      <%= f.collection_select :installation_ids, Installation.order(:name), :id, :name, 
```

Comment:
this line fails for me - as far as I can see there is no 'name' field in the installations table, it is a method defined in the Installation model, so ordering in the db cannot work